people.json

```diff
@@ -5,4 +5,12 @@
     "twitter": "@RamonGebben",
     "linkedin": "https: //www.linkedin.com/in/ramon-gebben-88039b85"
   }
+  
```

Comment:
It looks like you forgot to put a `,` before starting a new block.

Reply:
Done